README.md

```diff
@@ -550,7 +550,23 @@ server side.
 If writing your own gRPC clients in the language of choice consult
 gRPC guide on [cancellation](https://grpc.io/docs/guides/cancellation/#cancelling-an-rpc-call-on-the-client-side).
 
+### GRPC Status Codes
 
+Starting from release 24.09, Triton server introduces support for gRPC error codes in streaming mode for all clients enhancing error reporting capabilities. When this feature is enabled, the Triton server will return standard gRPC error codes and subsequently close the stream after delivering the error.
```

Comment:
For the sake of completion, can you also describe what happens when the header is not provided?
What is the default behaviour? That is the error objects being delivered in the callbacks.